### PR TITLE
BZ 1868676: document that RHCOS workers are expected

### PR DESCRIPTION
**Pull request by ousleyp**
Bug: https://bugzilla.redhat.com/show_bug.cgi?id=1868676

enterprise-4.5, 4.6

Preview: https://virt-coreos-workers--ocpdocs.netlify.app/openshift-enterprise/latest/virt/install/preparing-cluster-for-virt.html

### CODE REVIEW

**Comment by fabiand** [comment]
lgtm!

File: virt/install/preparing-cluster-for-virt.adoc [hunk 1]
```diff
@@ -4,15 +4,18 @@ include::modules/virt-document-attributes.adoc[]
 :context: preparing-cluster-for-virt
 toc::[]
```
Comment (aburdenthehand) [inline virt/install/preparing-cluster-for-virt.adoc:6]: This is now a Note followed by two bullet points. Should we have an introductory sentence?

**Pull request merged**